src/hank.erl

```diff
@@ -60,3 +58,6 @@ ignored_rules(AST, Rules) ->
            end
         end,
     erl_syntax_lib:fold(FoldFun, [], erl_syntax:form_list(AST)).
+
+analyze(Rules, ASTs, Context) ->
+    rpc:pmap({hank_rule, analyze}, [ASTs, Context], Rules).
```

Comment:
just to ensure https://github.com/AdRoll/rebar3_hank/pull/76 with this case, it will be useful to merge this in your branch @diegomanuel when it gets merged into main